best-practices.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+    env: {
+        browser: true,
+        node: true
+    },
+    rules: {
```

Comment:
I think we shouldn't include any environment setup in the rules themselves to make them more composable.
So like:
```javascript
module.exports = {
    rules: {
        /* ... */
    }
}